### PR TITLE
 feat: Added css and js for sum-app 

### PR DESCRIPTION
**Pull request by Dithinkrishna**

### CODE REVIEW

File: 1-sum-app/sum.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <script src="sum-app.js"></script>
+    </head>
+    <body>
+        <div class="wrapper-content">
+            <div class="input-wrapper">
+                <div>
+                 <u><h1 class="main-heading">Add Two Numbers </h1></u>
+                </div>
+                <div class="inp-row">
+                 <label>First number </label><input type="number" id="inp-text1" >
```
Comment (dinumathai) [inline 1-sum-app/sum.html]: Please give a meaningful name instead of `inp-text1`. Also for the other inputs.

File: 1-sum-app/sum-app.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
+.wrapper-content{
+
```
Comment (dinumathai) [inline 1-sum-app/sum-app.css]: Please avoid these extra spaces. I think we need to find some better css formatter plugin.

File: 1-sum-app/sum-app.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    text-align: center;
+
+}
+.input-wrapper{
+    width: 400px;
+    height: 350px;
+    margin: auto;
+
+    text-align: center;
+    margin-top: 28px;
```
Comment (dinumathai) [inline 1-sum-app/sum-app.css:10]: Always better to put the put similar CSS one below other. In this case `margin: auto;` and ` margin-top: 28px;` must be once below other.

File: 1-sum-app/sum.html (hunk 1 above)
Comment (Dithinkrishna) [reply 1-sum-app/sum.html]: used instead 
. inp-first-num
. input-second-num
. output-result

**Review by dinumathai — APPROVED** [review]

**Pull request closed without merging**